.pylintrc

```diff
@@ -4,45 +4,74 @@ disable=
  import-error,
```

Comment:
I'd rather not make .pylinrc changes part of this patch.
You have copied .pylintrc from leapp-repository, but they have initially been different, so I would fix existing issues and not enable import order check yet.
I fixed current linting violations in https://github.com/oamg/leapp/pull/768 , maybe we could use this one and you would remove .pylintrc changes from your pr?

Reply:
Well the previous version introduces also many problems, but since it is disabled I don't really care about it that much, I can drop it.